src/manipulation.js

```diff
@@ -26,7 +26,7 @@ define( [
 	dataPriv, dataUser, acceptData ) {
 
 var
-	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:-]+)[^>]*)\/>/gi,
+	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([a-z][^\s\/\0>]*)[^>]*)\/>/gi,
```

Comment:
See above regarding `\s` and sizing, but note that this one should accept `:`. Likewise on all counts for `rtagName` as well.